packages/junon-io/server/entities/remote_event_handler.js

```diff
@@ -4,59 +4,43 @@ const Protocol = require('../../common/util/protocol')
 const FirebaseAdminHelper = require("../util/firebase_admin_helper")
 const BadWordsFilter = require("../util/bad_words_filter")
 const SectorBanModel = require("junon-common/db/sector_ban")
-const xss = require("xss")
-
 class RemoteEventHandler {
-
   constructor(server) {
     this.server = server
   }
-
   getSocketUtil() {
     return this.server.socketUtil
   }
-
   onSocketMessage(eventName, data, socket) {
     let handlerName = "on" + eventName
 
     let player = this.getPlayerForSocket(socket)
 
-    const json = JSON.stringify(data.toJSON())
-    const xssfree = xss(json)
-    const cleanedData = JSON.parse(xssfree)
-
     let eventsNotRequiringPlayer = ["WindowResized", "RequestGame", "ResumeGame", "Ping"]
     if (eventsNotRequiringPlayer.indexOf(eventName) !== -1) {
-      this[handlerName](player, cleanedData, socket)
+      this[handlerName](player, data, socket)
     } else if (player) {
-      this[handlerName](player, cleanedData, socket)
+      this[handlerName](player, data, socket)
     }
 
   }
 
   onPing(player, data, socket) {
     this.getSocketUtil().emit(socket, "Pong", {})
   }
-
   onInviteToTeam(player, data, socket) {
     let targetPlayer = this.server.getPlayerById(data.playerId)
     if (!targetPlayer) return
-
     let team = player.getJoinableTeam()
     if (!team) return
   }
-
   onSendMoney(player, data, socket) {
     let targetPlayer = this.server.getPlayerById(data.playerId)
     if (!targetPlayer) return
-
```

Comment:
too many unecessary newline removal

Reply:
okay, I'll see if I can fix it, I'm not sure why github did that